### ISSUE TITLE
Can't connect to X1C that is in LAN only mode as long as Bambu studio is logged in to the online services.

### ISSUE THREAD

**Issue by kozmaz87** [bambulab/BambuStudio#5315]
### Bambu Studio Version

1.10.1.50

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 10

### Additional system information

Not relevant.

### Printer

X1 Carbon

### How to reproduce

Printer works in LAN only mode as long as Bambu Studio is logged out. 

1. Log into Bambu studio and printer no longer connects.
2. Try the add with access code wizard. It will not work. It will add the printer and you will be greeted with this screen
![image](https://github.com/user-attachments/assets/1832a87a-7c9f-4ce9-b546-9aec29eb23bd)
3. Log out from Bambu Studio on the home screen.
4. See your printer connected.
![image](https://github.com/user-attachments/assets/89da6211-b2aa-4a18-9656-8c3d348c0e72)

Furthermore I have some feeadback about your approach to LAN only mode: 
 * Firmware update does not work in LAN only mode. The device update page is blank.
 
![image](https://github.com/user-attachments/assets/a97b1f75-7c35-40b6-b2d5-af1de9252803)

 * The SD card contents should not be disabled in LAN only mode in Bambu Studio considering the things are a simple FTP connection away to the printer, which works outside Bambu Studio but I would prefer to have the full functionality inside it without having to push everything through Bambu's often malfunctioning and unavailable servers.

### Actual results

Printer does not connect

### Expected results

Printer should connect

### Project file & Debug log uploads

Project is not applicable it can be reproduced when untitled blank project is in view.
[log.zip](https://github.com/user-attachments/files/17791686/log.zip)

### Checklist of files to include

- [X] Log file
- [ ] Project file

**Comment by miikememe** [comment]
Yeah this is still the case, have to log out of online account for LAN printer to connect. 

SD Card is not available in LAN mode. This makes no sense.

If I reopen Bambu, I have to manually reselect my printer each time, this is frustratingly problematic and interruptive of workflow
